<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit01">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit02">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit03">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit04">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit05">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit06">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>目前目錄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>轉檔命令</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>取得清單</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>清單名稱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>來源路徑</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>輸出路徑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>已轉筆數</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit07">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
